nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/DistributedCacheServer.java

```diff
@@ -68,6 +68,13 @@
         .required(false)
         .addValidator(StandardValidators.createDirectoryExistsValidator(true, true))
         .build();
+    public static final PropertyDescriptor MAX_READ_SIZE = new PropertyDescriptor.Builder()
+        .name("Maximum Read Size")
```

Comment:
A `displayName` should also be included:
```suggestion
        .name("maximum-read-size")
        .displayName("Maximum Read Size")
```

Reply:
Cool; now I understand the PR template instructions.  TIL!